### PR TITLE
Login page for ecommerce

### PR DESCRIPTION
**Pull request by vashnavichauhan1825**

### CODE REVIEW

File: login.css [hunk 1]
```diff
@@ -0,0 +1,188 @@
+@import url('https://vaspacx-components.netlify.app/style.css');
+@import url('https://fonts.googleapis.com/css2?family=Manrope:wght@200;300;400;500;600&display=swap');
+
+:root{
+ 
+    --mainBrandColor: #3a7ce3;
```
Comment (vrx29) [inline login.css:6]: better naming can be done for variables. Generally, the camelCase naming convention is a bad practice in CSS. Try to separate them with '-'.

**Comment by VLeads** [comment]
Code looks good. But a suggestion that you should commit code at once. It will give ease while reviewing.

File: login.css [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+    margin: 0;
+}
+
+.login-box{
+    box-sizing: border-box;
+    color:var(--font-primary);
+    padding: 0;
+    background-color: var(--secondaryDark);
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
```
Comment (b0llu) [inline login.css]: you can use grid-template-columns: repeat(3, 1fr);

File: login.css [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+    padding: 0px 30px 20px 30px;
+}
+
+.container-login label{
+    display: inline-block;
+    margin-bottom: 4px;
+    margin-top: 15px;
+}
+
+
```
Comment (b0llu) [inline login.css:70]: remove extra spacing

File: login.css [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+}
+
+.vaspacx{
+    font-family: 'Roboto', sans-serif;
+    font-size: 74px;
+    letter-spacing: 2px;
+    padding-bottom: 23px;
+    background: -webkit-linear-gradient(
+349deg
+,var(--logo-color) 39%, #3a7ce3 29%);
```
Comment (b0llu) [inline login.css]: the code can look better here with a code formatter just a suggestion

File: login.css [hunk 5]
```diff
@@ -0,0 +1,198 @@
...
+    display:flex;
+    justify-content:center;
+    gap:20%;
+    align-items:center;
+   
+   
+}
+.login-logo-container{
+    width: 100%;
+    /* background-color: blue; */
```
Comment (b0llu) [inline login.css]: remove commented code

File: signin.html [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+    var icon = document.getElementById("icon-login");
+
+    icon.onclick = function(){
+      document.body.classList.toggle("light-theme");
+      if(document.body.classList.contains("light-theme")){
+        icon.src="./img/moon.png";
+      }else{
+        icon.src = "./img/sun.png"
+      }
+    }
```
Comment (b0llu) [inline signin.html:59]: its better to create a different JS file rather than writing js in html itself

Comment (vashnavichauhan1825) [reply signin.html:59]: it's few line of code that'y I put it in same file if the code line increases I will definitely put it in another file

File: login.css (hunk 5 above)
Comment (vashnavichauhan1825) [reply login.css]: sure

File: login.css (hunk 4 above)
Comment (vashnavichauhan1825) [reply login.css]: ok rectify it

File: login.css (hunk 2 above)
Comment (vashnavichauhan1825) [reply login.css]: yes thankyou forgot about that property

**Pull request merged**